### PR TITLE
Porting modifications from 74

### PR DESCRIPTION
**Pull request by dinardo**
Porting modifications from 74

### CODE REVIEW

**Comment by deguio** [comment]
-1
here apparently a rebase is needed.
thanks,
F.

**Comment by dinardo** [comment]
Hi,
what are the problems here ?
Is there an action required by me ?
Do I need to change something in the code ?

Many thanks for letting me know.
- Mauro.

File: DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -30,14 +33,17 @@
 process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
 process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
 process.load("Configuration.StandardSequences.RawToDigi_Data_cff")
-process.load("DQM.Integration.config.FrontierCondition_GT_cfi")
```
Comment (deguio) [inline DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py]: this is causing a crash on the playback where we are testing this PR.
please revert this line back and propagate the fix to 75 and 76 as well.

**Comment by deguio** [comment]
ciao mauro,
this PR is not mergeable anymore and you need to rebase it on top of the CMSSW_7_4_X branch.
moreover a fix is needed (see my comments above). you cannot access frontier 'a la offline' from inside P5.

**Comment by dinardo** [comment]
OK done.
I've rolled back as you suggested.
Sorry about that,

- Mauro.

**Comment by dinardo** [comment]
Hold on this.
I still need to do the re-base stuff ... which I don't know how to do it.
- Mauro.

**Comment by dinardo** [comment]
Hi, I preferred to make a new brand new pull request instead of rebasing this one.
The new pull request is : #11256
Please discard this pull request.
- Mauro.

**Pull request closed without merging**